.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## PR Description: *short_description*
+
+### Tickets:
+ - *ticket-id*
```

Comment:
Can we add tickets from private bug tracer here?

Reply:
I think we can add the bug id